### PR TITLE
Logstash

### PR DESCRIPTION
**Pull request by emmanuellyautomated**
points logstash submodule to [this fork](https://github.com/emmanuellyautomated/ansible-role-logstash/) which contains 5C templates. Client-specific data can be found [here](https://github.com/5thColumn/stackboss/pull/1)

### CODE REVIEW

**Comment by vforgione** [comment]
@emmanuellyautomated, is this still a good PR? If so, can you merge in master and resolve the conflicts?

**Comment by emmanuellyautomated** [comment]
@vforgione, this PR is now good. Would you mind giving it a "review".

File: .gitmodules [hunk 1]
```diff
@@ -39,7 +39,7 @@
 	url = git@github.com:geerlingguy/ansible-role-kibana.git
 [submodule "logstash"]
 	path = logstash
-	url = git@github.com:geerlingguy/ansible-role-logstash.git
+	url = https://github.com/emmanuellyautomated/ansible-role-logstash
```
Comment (vforgione) [inline .gitmodules]: Why is this `https` instead of `git`?

**Comment by emmanuellyautomated** [comment]
I'm just going to 🔥 this down...

**Pull request closed without merging**